docs/dev/how-to-write-and-run-tests.md

```diff
@@ -64,7 +75,6 @@ To run a single test you can use:
    ```bash
    make int-test test="filename.t"
    ```
-
```

Comment:
The documentation is incorrect, make unit-test test="filename.t"  should be make test-unit test="filename.t"

Same for make int-test test="filename.t" which should be make test-int test="filename.t"